packages/desktop-client/src/browser-preload.browser.js

```diff
@@ -121,9 +121,9 @@ global.Actual = {
   openURLInBrowser: url => {
     window.open(url, '_blank');
   },
-  onEventFromMain: (type, handler) => {},
+  onEventFromMain: () => {},
   applyAppUpdate: () => {},
-  updateAppMenu: isBudgetOpen => {},
+  updateAppMenu: () => {},
```

Comment:
Several of these preload functions match types used in the electron preload, so these parameters whilst not used here are used in the electron implementation of these functions.
Personally I'd he hesitant to change any of these as its quite convenient that they have the same interface in both worlds.

Reply:
What do you suggest? Ignoring the rule for those cases maybe?
Feel free to make any edits you consider necessary to the PR.